Flaky TestLtrMsmarcoDocument test

Ref: #951

```
======================================================================
FAIL: test_reranking (integrations.sparse.test_lucenesearcher_check_ltr_msmarco_document.TestLtrMsmarcoDocument)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jimmylin/workspace/pyserini/integrations/sparse/test_lucenesearcher_check_ltr_msmarco_document.py", line 54, in test_reranking
    self.assertAlmostEqual(float(mrr), 0.3108, delta=0.0001)
AssertionError: 0.3109181374378502 != 0.3108 within 0.0001 delta (0.00011813743785016317 difference)

----------------------------------------------------------------------
Ran 1 test in 7426.656s
```

Minor differences on my Mac Studio (M1) after upgrade of Python env.

--- a/integrations/sparse/test_lucenesearcher_check_ltr_msmarco_document.py
+++ b/integrations/sparse/test_lucenesearcher_check_ltr_msmarco_document.py
@@ -50,8 +50,10 @@ class TestLtrMsmarcoDocument(unittest.TestCase):
         result = subprocess.check_output(f'python tools/scripts/msmarco/msmarco_doc_eval.py --judgments tools/topics-and-qrels/qrels.msmarco-doc.dev.txt --run ltr_test/{outp}', shell=True).decode(sys.stdout.encoding)
         a,b = result.find('#####################\nMRR @100:'), result.find('\nQueriesRanked: 5193\n#####################\n')
         mrr = result[a+32:b]
-        # See https://github.com/castorini/pyserini/issues/951
-        self.assertAlmostEqual(float(mrr), 0.3108, delta=0.0001)
+        # See:
+        #  - https://github.com/castorini/pyserini/issues/951
+        #  - https://github.com/castorini/pyserini/issues/1430
+        self.assertAlmostEqual(float(mrr), 0.3108, delta=0.0002)
         rmtree('ltr_test')
 
 if __name__ == '__main__':

